app/javascript/components/MappingWizard/index.js

```diff
@@ -2,84 +2,39 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { FormattedMessage } from 'react-intl';
-import {
-  Button,
-  Icon,
-  EmptyState,
-  Modal,
-  Spinner,
-  Wizard
-} from 'patternfly-react';
+import { noop, selectKeys } from '../../common/helpers';
 import * as MappingWizardActions from '../../redux/actions/mappingWizard';
-import { noop } from '../../common/helpers';
-
-const loadingContents = () => (
-  <EmptyState>
-    <Spinner size="lg" className="blank-slate-pf-icon" loading />
-    <EmptyState.Action>
-      <h3>Loading Wizard</h3>
-    </EmptyState.Action>
-    <EmptyState.Action secondary>
-      <p>
-        Lorem ipsum dolor sit amet, porta at suspendisse ac, ut wisi vivamus,
-        lorem sociosqu eget nunc amet.{' '}
-      </p>
-    </EmptyState.Action>
-  </EmptyState>
-);
+import ModalWizard from '../ModalWizard';
+import MappingWizardBody from './MappingWizardBody';
 
 class MappingWizardContainer extends React.Component {
   componentDidMount() {
     const { url, fetchSourceClusters } = this.props;
 
     fetchSourceClusters(url);
   }
+
   render() {
-    const { showWizard, onHide, onExited, sourceClusters } = this.props;
+    const { sourceClusters } = this.props;
+    const loaded = sourceClusters && sourceClusters.length > 0;
```

Comment:
Redux is already injecting `isFetching` inside `this.props` here so i think we can just use that instead...

Reply:
Ok. Done.